### ISSUE TITLE
Upgrade from `C118` -> `C119` on Win x64 via 1.61.x - Nightly (Checking C119)

### ISSUE THREAD

**Issue by kjozwiak** [brave/brave-browser#33767]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [x] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [x] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [x] Remove the following component folders and ensure that they're being re-downloaded after restarting the browser:
  - [x] `afalakplffnnnlkncjhbmahjfjhmlkal`: `AutoplayWhitelist.dat`, `ExtensionWhitelist.dat`, `ReferrerWhitelist.json` and `Greaselion.json`
  - [x] `CertificateRevocation`
  - [x] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
  - [x] `gkboaolpopklhgplhaaiboijnklogmbc`: Brave Ad Block List Catalog
  - [x] `iodkpdagapdfkphljnddpjlldadblomo`: Brave Ad Block Updater
  - [x] `mfddibmblmbccpadfndgakiopmmhebop`: Brave Ad Block List Catalog
  - [x] `oofiananboodjbbmdelgdommihjbkfag`: HTTPSE
  - [x] `Safe Browsing`
- [x] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers

### Upgrade

- [x] Make sure that data from the last version appears in the new version OK
- [x] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [x] With data from the last version, verify that:
  - [x] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [x] Cookies are preserved
  - [x] Installed extensions are retained and work correctly
  - [x] Opened tabs can be reloaded
  - [x] Stored passwords are preserved
  - [x] Sync chain created in previous version is retained
  - [x] Social media-blocking buttons changes are retained
  - [x] Custom filters under brave://settings/shields/filters are retained
  - [x] Custom lists under brave://settings/shields/filters are retained
  - [x] Rewards
    - [x] BAT balance is retained
    - [x] Auto-contribute list is retained
    - [x] Both Tips and Monthly Contributions are retained
    - [x] Panel transactions list is retained
    - [x] Changes to rewards settings are retained
    - [x] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [x] Ads
    - [x] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [x] Changes to ads settings are retained
    - [x] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [x] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 10 x64` using the following build(s):

```
Brave | 1.61.30 Chromium: 118.0.5993.70 (Official Build) nightly (64-bit)
-- | --
Revision | 7e4b32142a514fc5de17047cb19c5b6c06c00105
OS | Windows 10 Version 22H2 (Build 19045.3570)
```

<details>
<summary>C118 --> C119 on-demand upgrade check - PASSED</summary>

* installed `1.61.30 Chromium: 118.0.5993.70` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `cr-bump-update-tests-x64`
* realunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.61.39 Chromium: 119.0.6045.33` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|----------------|----------------
![image](https://github.com/brave/brave-browser/assets/2602313/2c0e340f-d452-4837-86b8-1e28db4d9701) | ![image](https://github.com/brave/brave-browser/assets/2602313/c568aad9-4a8e-407f-89cd-c7cdc20a47fd) | ![image](https://github.com/brave/brave-browser/assets/2602313/5fa7924f-2adf-4c9a-9a12-d8b2be19f5e8) | ![image](https://github.com/brave/brave-browser/assets/2602313/4c4aeff6-44e3-4ffd-847f-b6c579e4c3cf) | ![image](https://github.com/brave/brave-browser/assets/2602313/53094dfd-0c5f-449d-8805-5dfa4fcb1914)

</details>

<details>
<summary>C119 --> C119 on-demand upgrade check - PASSED</summary>

* installed `1.61.36 Chromium: 119.0.6045.33` via `BraveBrowserStandaloneNightlySetup_119_1_61_36.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `cr-bump-update-tests-x64`
* realunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.61.39 Chromium: 119.0.6045.33` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|---------------
![image](https://github.com/brave/brave-browser/assets/2602313/48ec4a78-80cf-4fdc-8e74-0ceac0f1181d) | ![image](https://github.com/brave/brave-browser/assets/2602313/23f0c55c-726c-44c6-8e1d-904d38923964) | ![image](https://github.com/brave/brave-browser/assets/2602313/b28c7b54-bee9-411b-8c20-9467979a19e9) | ![image](https://github.com/brave/brave-browser/assets/2602313/60d5f100-e661-4c6f-94f4-46348c0753a4)

```
[10/23/23 12:43:31.118][BraveUpdate:goopdate][9712:1600][Installing][display name: Brave Nightly][app id: {C6CB981E-DB30-4876-8639-109F8933582C}][installer path: C:\Program Files (x86)\BraveSoftware\Update\Install\{FB009187-A6D5-4F96-A835-5EEBEF0F752C}\brave_installer-delta-x64.exe][manifest args: --chrome-sxs][installer data: ][untrusted data: ][priority: 10]
[10/23/23 12:43:31.119][BraveUpdate:goopdate][9712:1600][Running installer][C:\Program Files (x86)\BraveSoftware\Update\Install\{FB009187-A6D5-4F96-A835-5EEBEF0F752C}\brave_installer-delta-x64.exe][--chrome-sxs][{C6CB981E-DB30-4876-8639-109F8933582C}]
[10/23/23 12:43:49.232][BraveUpdate:goopdate][9712:1600][InstallerResult][{C6CB981E-DB30-4876-8639-109F8933582C}][0]
[10/23/23 12:43:49.232][BraveUpdate:goopdate][9712:1600][InstallApp returned][0x0][{C6CB981E-DB30-4876-8639-109F8933582C}][type:1][code: 30][][]
[10/23/23 12:43:49.291][BraveUpdate:goopdate][9712:1600][Started process][6384]
[10/23/23 12:43:49.306][BraveCrashHandler][6384:2444][ConfigManager::LoadGroupPolicies][Machine is not Enterprise Managed]
[10/23/23 12:43:49.307][BraveCrashHandler][6384:2444][ConfigManager::LoadGroupPolicies][No Group Policies found under key][HKLM\Software\Policies\BraveSoftware\Update\]
[10/23/23 12:43:49.307][BraveCrashHandler][6384:2444][OmahaPolicyManager::set_policy][Group Policy][[CachedOmahaPolicy][is_initialized][0][is_managed][0][auto_update_check_period_minutes][-1][download_preference][][cache_size_limit][-1][cache_life_limit][-1][updates_suppressed][-1][-1][-1][proxy_mode][][proxy_server][][proxy_pac_url][][install_default][-1][update_default][-1]]
[10/23/23 12:43:49.310][BraveCrashHandler][6384:2444][CrashHandler::RunUntilShutdown]
[10/23/23 12:43:49.340][BraveUpdate:goopdate][9712:1600][Started process][13132]
[10/23/23 12:43:49.350][BraveUpdate:goopdate][10952:8788][DllEntry exit][0x00000000]
```

</details>

<details>
<summary>C118 --> C119 background upgrade check - PASSED</summary>

* installed `1.61.30 Chromium: 118.0.5993.70` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `cr-bump-update-tests-x64`
* started `BraveSoftwareUpdateTaskMachineUA{8BCB450B-FEF6-4528-B38C-0988EC8753F9}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.61.39 Chromium: 119.0.6045.33`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background`

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|---------------|----------------
![image](https://github.com/brave/brave-browser/assets/2602313/ccb611bb-1e98-4513-a93c-7b484da49ce7) | ![image](https://github.com/brave/brave-browser/assets/2602313/bbf7f5d1-9243-4499-bdad-8205c84907cb) | ![image](https://github.com/brave/brave-browser/assets/2602313/8a88bd15-3de6-4a42-8dc4-4d2a88d5c23d) | ![image](https://github.com/brave/brave-browser/assets/2602313/7a0cdc7c-f3e8-4305-9155-13f61dace2f7)

</details>

<details>
<summary>C119 --> C119 background upgrade check - PASSED</summary>

* installed `1.61.36 Chromium: 119.0.6045.33` via `BraveBrowserStandaloneNightlySetup_119_1_61_36.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `cr-bump-update-tests-x64`
* started BraveSoftwareUpdateTaskMachineUA{A1B676DC-A4FA-4B4D-B13F-AFBBDDA6B0E9}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.61.39 Chromium: 119.0.6045.33`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|----------------
![image](https://github.com/brave/brave-browser/assets/2602313/3456c8dc-0e31-4cb0-880d-a7857d00391c) | ![image](https://github.com/brave/brave-browser/assets/2602313/0b2d5c3b-834a-4ef0-bc6b-f1ab673d7d2e) | ![image](https://github.com/brave/brave-browser/assets/2602313/38ebfe1f-07ae-4bdc-87dd-257943a088e1) | ![image](https://github.com/brave/brave-browser/assets/2602313/1225d91e-84f6-4ae2-883b-7a376f994d33)

</details>

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 10 x64` using the following build(s):

```
Brave | 1.61.32 Chromium: 118.0.5993.88 (Official Build) nightly (32-bit)
-- | --
Revision | 8b2743769e6138ec51e617fd7641a9b0171a9172
OS | Windows 10 Version 22H2 (Build 19045.3570)
```

<details>
<summary>C118 --> C119 on-demand upgrade check - PASSED</summary>

* installed `1.61.32 Chromium: 118.0.5993.88` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `cr-bump-update-tests-x86`
* realunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.61.39 Chromium: 119.0.6045.33` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|----------------|----------------
![image](https://github.com/brave/brave-browser/assets/2602313/530bdb7a-fe47-44ae-ac30-6345e04d2488) | ![image](https://github.com/brave/brave-browser/assets/2602313/c9ff76fd-5601-457d-9cdb-4f14263297d1) | ![image](https://github.com/brave/brave-browser/assets/2602313/216efe6b-e7cb-478c-9b4f-46b2bd91f393) | ![image](https://github.com/brave/brave-browser/assets/2602313/2fa670cb-b4ba-41a9-8854-5292a440ccba) | ![image](https://github.com/brave/brave-browser/assets/2602313/56af81f6-3e2b-4e1b-bffa-3e625d226e3a)

</details>

<details>
<summary>C119 --> C119 on-demand upgrade check - PASSED</summary>

* installed `1.61.36 Chromium: 119.0.6045.33` via `BraveBrowserStandaloneNightlySetup32_119_1_61_36.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `cr-bump-update-tests-x86`
* realunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.61.39 Chromium: 119.0.6045.33` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|----------------|-----------------
![image](https://github.com/brave/brave-browser/assets/2602313/c2a8b41a-38f8-4e7c-bf96-567f729694fc) | ![image](https://github.com/brave/brave-browser/assets/2602313/915432b4-801c-4cad-afe3-a4292b3b2512) | ![image](https://github.com/brave/brave-browser/assets/2602313/48d63dc1-0d23-4846-9fba-f754c17d3054) | ![image](https://github.com/brave/brave-browser/assets/2602313/b2c0bb99-8552-4603-a0fc-844c92fcf370) | ![image](https://github.com/brave/brave-browser/assets/2602313/8dc09829-4b58-4403-bfd1-f17d10a1eca3)

```
[10/23/23 15:05:31.904][BraveUpdate:goopdate][11860:12284][Installing][display name: Brave Nightly][app id: {C6CB981E-DB30-4876-8639-109F8933582C}][installer path: C:\Program Files\BraveSoftware\Update\Install\{E77C4D16-A474-4D1A-A1C0-38A03644AB17}\brave_installer-delta-ia32.exe][manifest args: --chrome-sxs][installer data: ][untrusted data: ][priority: 10]
[10/23/23 15:05:31.904][BraveUpdate:goopdate][11860:12284][Running installer][C:\Program Files\BraveSoftware\Update\Install\{E77C4D16-A474-4D1A-A1C0-38A03644AB17}\brave_installer-delta-ia32.exe][--chrome-sxs][{C6CB981E-DB30-4876-8639-109F8933582C}]
[10/23/23 15:05:49.637][BraveUpdate:goopdate][11860:12284][InstallerResult][{C6CB981E-DB30-4876-8639-109F8933582C}][0]
[10/23/23 15:05:49.637][BraveUpdate:goopdate][11860:12284][InstallApp returned][0x0][{C6CB981E-DB30-4876-8639-109F8933582C}][type:1][code: 30][][]
[10/23/23 15:05:49.640][BraveUpdate:goopdate][11860:12284][Started process][11460]
[10/23/23 15:05:49.648][BraveCrashHandler][11460:12196][ConfigManager::LoadGroupPolicies][Machine is not Enterprise Managed]
[10/23/23 15:05:49.649][BraveCrashHandler][11460:12196][ConfigManager::LoadGroupPolicies][No Group Policies found under key][HKLM\Software\Policies\BraveSoftware\Update\]
[10/23/23 15:05:49.649][BraveCrashHandler][11460:12196][OmahaPolicyManager::set_policy][Group Policy][[CachedOmahaPolicy][is_initialized][0][is_managed][0][auto_update_check_period_minutes][-1][download_preference][][cache_size_limit][-1][cache_life_limit][-1][updates_suppressed][-1][-1][-1][proxy_mode][][proxy_server][][proxy_pac_url][][install_default][-1][update_default][-1]]
[10/23/23 15:05:49.650][BraveCrashHandler][11460:12196][CrashHandler][Instance is already running][32-bit][1]
[10/23/23 15:05:49.813][BraveUpdate:goopdate][2924:4956][DllEntry exit][0x00000000]
```

</details>

<details>
<summary>C118 --> C119 background upgrade check - PASSED</summary>

* installed `1.61.32 Chromium: 118.0.5993.88` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `cr-bump-update-tests-x86`
* started `BraveSoftwareUpdateTaskMachineUA{86856C14-FE7D-49C1-B36E-6B5DE5F66120}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.61.39 Chromium: 119.0.6045.33`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background`

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|---------------|-----------------
![image](https://github.com/brave/brave-browser/assets/2602313/06727e4d-6a89-456a-8741-a21e757c7cd6) | ![image](https://github.com/brave/brave-browser/assets/2602313/c046e275-ea0d-4793-bfb9-be3902d725c5) | ![image](https://github.com/brave/brave-browser/assets/2602313/7bfe0a0d-ea9e-4b4e-9cf1-499d1b336bc4) | ![image](https://github.com/brave/brave-browser/assets/2602313/c7ccebc5-5a83-448c-b50a-38763356ccff)

</details>

<details>
<summary>C119 --> C119 background upgrade check - PASSED</summary>

* installed `1.61.36 Chromium: 119.0.6045.33` via `BraveBrowserStandaloneNightlySetup32.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `cr-bump-update-tests-x86`
* started `BraveSoftwareUpdateTaskMachineUA{3A8E4C01-1D47-4BED-BA10-99070ACE9BE4}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.61.39 Chromium: 119.0.6045.33`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|----------------
![image](https://github.com/brave/brave-browser/assets/2602313/c77576b7-22e8-4381-b2b3-6dd29c7172b4) | ![image](https://github.com/brave/brave-browser/assets/2602313/5bb52ba5-2a3c-4778-a187-ffd960268b67) | ![image](https://github.com/brave/brave-browser/assets/2602313/af8207ed-6363-47f4-af80-3ccaa756496c) | ![image](https://github.com/brave/brave-browser/assets/2602313/9456657e-ad69-4135-a518-241c1bf644c1)

</details>
